main.py

```diff
@@ -38,6 +43,8 @@
             template_folder='./frontend/dist')
 # Set flask logging to warning
 logging.getLogger('werkzeug').setLevel(logging.WARNING)
+# Set paramiko logging to warning
+logging.getLogger('paramiko').setLevel(logging.WARNING)
```

Comment:
We are not using `paramiko` in this project, so we don't need to change it's logging level.